src/utils/coordinates.js

```diff
@@ -1,5 +1,5 @@
 /* global THREE */
-
+var extend = require('object-assign');
```

Comment:
Should probably move all the object utils (extend, deep extend, diff, clone) to a separate module in the future.